django/thunderstore/api/cyberstorm/views/user.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+from rest_framework.exceptions import PermissionDenied, ValidationError
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.api.utils import conditional_swagger_auto_schema
+from thunderstore.social.views import DeleteAccountForm
+
+User = get_user_model()
```

Comment:
Why is this here?

Reply:
Left over, weird that IDE didn't complain about it.